values.yaml

```diff
@@ -1073,4 +1077,4 @@ external-dns:
   provider: aws
   aws:
     zoneType: "public"
-  domainFilters: []
+  domainFilters: []
```

Comment:
please keep the newline at the end of file

Reply:
New line added!